label_studio/tasks/openapi_schema.py

```diff
@@ -34,8 +34,8 @@
     'comment_count': 0,
     'unresolved_comment_count': 0,
     'last_comment_updated_at': '2024-01-15T09:30:00Z',
```

Comment:
I believe it should be like this: 
`2024-06-18T23:45:46.048490Z`

pls, fix `created_at` and `updated_at` too.

Reply:
Or maybe it's better to use the full version of api/tasks response: 
```
{
  "id": 13,
  "predictions": [],
  "annotations": [],
  "drafts": [],
  "annotators": [],
  "inner_id": 2,
  "cancelled_annotations": 0,
  "total_annotations": 0,
  "total_predictions": 0,
  "completed_at": null,
  "annotations_results": "",
  "predictions_results": "",
  "predictions_score": null,
  "file_upload": "6b25fc23-some_3.mp4",
  "storage_filename": null,
  "annotations_ids": "",
  "predictions_model_versions": "",
  "avg_lead_time": null,
  "draft_exists": false,
  "updated_by": [],
  "data": {
    "image": "/data/upload/1/6b25fc23-some_3.mp4"
  },
  "meta": {},
  "created_at": "2024-06-18T23:45:46.048490Z",
  "updated_at": "2024-06-18T23:45:46.048538Z",
  "is_labeled": false,
  "overlap": 1,
  "comment_count": 0,
  "unresolved_comment_count": 0,
  "last_comment_updated_at": null,
  "project": 1,
  "comment_authors": []
}
```

Reply:
Fixed